Can't compile quippy

`make quippy` fails (gcc/gfortran 9.4.0, python 3.9.18).  Errors start with
```
INFO: compile options: '-DNPY_DISABLE_OPTIMIZATION=1 -I./src.linux-x86_64-3.9/./src.linux-x86_64-3.9 -I/home/Software/python/system/extra/lib64/python3.9/site-packages/numpy/core/include -I/usr/include/python3.9 -c'
INFO: gcc: ./src.linux-x86_64-3.9/_quippymodule.c
./src.linux-x86_64-3.9/_quippymodule.c: In function ‘f2py_rout__quippy_f90wrap_table_pointer_initialise’:
./src.linux-x86_64-3.9/_quippymodule.c:2273:11: error: ‘capi_this_tmp’ undeclared (first use in this function); did you mean ‘capi_this_intent’?
 2273 |       if (capi_this_tmp == NULL) {
      |           ^~~~~~~~~~~~~
      |           capi_this_intent
./src.linux-x86_64-3.9/_quippymodule.c:2273:11: note: each undeclared identifier is reported only once for each function it appears in
./src.linux-x86_64-3.9/_quippymodule.c:2343:18: error: invalid storage class for function ‘f2py_rout__quippy_f90wrap_table_pointer_finalise’
 2343 | static PyObject *f2py_rout__quippy_f90wrap_table_pointer_finalise(const PyObject *capi_self,
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./src.linux-x86_64-3.9/_quippymodule.c: In function ‘f2py_rout__quippy_f90wrap_table_pointer_finalise’:
./src.linux-x86_64-3.9/_quippymodule.c:2447:18: error: invalid storage class for function ‘f2py_rout__quippy_f90wrap_connection__get__initialised’
 2447 | static PyObject *f2py_rout__quippy_f90wrap_connection__get__initialised(const PyObject *capi_self,
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./src.linux-x86_64-3.9/_quippymodule.c: In function ‘f2py_rout__quippy_f90wrap_connection__get__initialised’:
.
.
.
```
followed by hundreds more.

f90wrap says it's 0.2.8 (which should be OK according to the Makefile), f2py-f90wrap says 1.26.2 and patched from f90wrap by JRK.

When I try to `make quippy`, I get the error

```
Traceback (most recent call last):
  File "/Users/albert/Library/Python/3.12/bin/f2py-f90wrap", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/albert/Library/Python/3.12/lib/python/site-packages/f90wrap/scripts/f2py_f90wrap.py", line 219, in main
    numpy.f2py.main()
  File "/Users/albert/Library/Python/3.12/lib/python/site-packages/numpy/f2py/f2py2e.py", line 766, in main
    run_compile()
  File "/Users/albert/Library/Python/3.12/lib/python/site-packages/numpy/f2py/f2py2e.py", line 738, in run_compile
    builder.compile()
  File "/Users/albert/Library/Python/3.12/lib/python/site-packages/numpy/f2py/_backends/_meson.py", line 176, in compile
    self.sources = _prepare_sources(self.modulename, self.sources, self.build_dir)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/albert/Library/Python/3.12/lib/python/site-packages/numpy/f2py/_backends/_meson.py", line 188, in _prepare_sources
    shutil.copy(source, bdir)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/shutil.py", line 435, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/shutil.py", line 240, in copyfile
    raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
shutil.SameFileError: 'f90wrap_Atoms_types.f90' and './f90wrap_Atoms_types.f90' are the same file
make[1]: *** [Makefile:118: quippy/_quippy.cpython-312-darwin.so] Error 1
make[1]: Leaving directory '/Users/albert/Code/QUIP_test/build/darwin_x86_64_gfortran'
make: *** [Makefile:230: quippy] Error 2
```

My `f90wrap` is version 0.2.13

--- a/quippy/Makefile
+++ b/quippy/Makefile
@@ -86,7 +86,7 @@ F2PY_LINK_ARGS = $(shell ${PYTHON} -c 'import sys; print(" ".join([arg for arg i
 all: build
 
 f90wrap:
-	${PIP} install ${QUIPPY_INSTALL_OPTS} "f90wrap>=0.2.6,<0.2.14"
+	${PIP} install ${QUIPPY_INSTALL_OPTS} "f90wrap==0.2.13"
 
 clean:
 	rm -f _quippy${EXT_SUFFIX} ${F90WRAP_FILES} ${WRAP_FPP_FILES}

